tests/src/astro/orbit_determination/observation_partials/unitTestRotationMatrixPartials.cpp

```diff
@@ -539,7 +541,7 @@ BOOST_AUTO_TEST_CASE( testIauRotationPartials )
                               << std::endl
                               << rotationMatrixPartials.at( librationIndex ) << std::endl
                               << std::endl
-                              << matrixDifference.cwiseQuotient( rotationMatrixPartials.at( librationIndex ) ) << std::endl
```

Comment:
Could you also re-enable the test statement below this, with a suitable tolerance?